.eslintrc.js

```diff
@@ -1,19 +1,11 @@
 module.exports = {
-  parser: '@babel/eslint-parser',
-  parserOptions: {
-    requireConfigFile: false,
-    babelOptions: {
-      presets: ['next/babel'],
```

Comment:
Are we able to remove the various babel/eslint dependencies from our `package.json`?

Reply:
no i don't think we should remove it from our package.json